### DISCUSSION TITLE
Enable https to a subdomain pointing outside Vercel

### DISCUSSION THREAD

**Discussion by jamesagwa** [vercel/community#3301] [Feature Question] [answered]
### Summary

After following this guide [https://vercel.com/guides/pointing-subdomains-to-external-services](https://vercel.com/guides/pointing-subdomains-to-external-services) to point my subdomain outside Vercel. I noticed the domain doesn't work on `https` only on `http`. Also I get an Invalid Congifuration error on the DNS UI . How do I fix this? 

### Steps to Reproduce

Note that I am trying to make my subdomain an api base url. so I added an AWS Elastic IP of my EC2 instance to Vercel DNS config.

![Screenshot 2023-07-19 at 16 54 00](https://github.com/vercel/community/assets/11221356/6707ea28-3d14-4755-9517-0ddcd84788e3)

**Comment by swarnava** [comment, marked as answer]
Hi,

Unfortunately, we can only provide certificate for Vercel hosted website. You need to setup custom certificate on the webserver where you are hosting your website currently
